azext_iot/digitaltwins/command_map.py

```diff
@@ -200,3 +204,12 @@ def load_digitaltwins_commands(self, _):
         cmd_group.wait_command(
             "wait", "wait_private_endpoint_conn"
         )
+
+    with self.command_group(
+        "dt jobs import",
```

Comment:
We usually refer to resources or API entities as singular...i.e. 'job' instead of 'jobs'

We should also a root command group for `az dt job` exists (with help).